src/style.css

```diff
@@ -0,0 +1,135 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body,
+h2,
+p {
+  margin: 0;
+  padding: 0;
+
+  --main-accent-color: #060b35;
+  --secondary-color: #616070;
```

Comment:
Remove these variables to `html` tag or `::root`

Reply:
Fixed